### ISSUE TITLE
After successfull execution of test scripts Extent report is showing "Unknown" instead of "PASS" or "FAIL".

### ISSUE THREAD

**Issue by rajay458** [anshooarora/extentreports-java#998]
I am still getting "Unknown" tag in report. Dont know what am missing, can someone help me with this??

**My code:**

```
package report;
import org.testng.annotations.Test;
import com.relevantcodes.extentreports.ExtentReports;
import com.relevantcodes.extentreports.ExtentTest;
import com.relevantcodes.extentreports.ExtentTestInterruptedException;
import junit.framework.Assert;
import org.testng.annotations.BeforeSuite;
import java.io.File;
import java.lang.reflect.Method;
import org.testng.annotations.AfterMethod;
import org.testng.annotations.AfterSuite;
import org.testng.annotations.BeforeMethod;

public class NewTest {
	public static ExtentReports extent;
	public static ExtentTest test;
	public static ExtentTestInterruptedException testexception;
  @Test
  public void f() {
	  System.out.println("@Test 2");
	  Assert.assertEquals(true, true);
  }
  @Test
  public void f1() {
	  System.out.println("@Test 1");
	  Assert.assertEquals(true, true);
  }
  @BeforeMethod(groups = { "Sanity" })
	public  void beforeMethod(Method method) 
	{
		test = extent.startTest( (this.getClass().getSimpleName() +" :: "+  method.getName()),method.getName()); //Test Case Start Here
		test.assignAuthor("Keshav Kashyap"); //Test Script Author Name
		test.assignCategory("Sanity  ::  :: API VERSION - "); //Test Category Defined Here
	}
  @BeforeSuite
  public void beforeSuite() {
	  System.out.println("@BeforeSuite");
	//Report Directory and Report Name
			extent = new ExtentReports("Reports//api_execution_report.html", true); //Provide Desired Report Directory Location and Name
			extent.loadConfig(new File("Sanity.xml")); //Supporting File for Extent Reporting
			extent.addSystemInfo("Environment","QA-Sanity"); //It will provide Execution Machine Information
  }
  @AfterMethod(groups = { "Sanity" })
	public void afterMethod() 
	{
	  System.out.println("@AfterMethod");
		extent.endTest(test);
		extent.flush();	
	}
  @AfterSuite
  public void afterSuite() {
	  System.out.println("@AfterSuite");
	  extent.close();  // close the Test Suite
  }
}

```

Dependency:

		<dependency>
			<groupId>com.relevantcodes</groupId>
			<artifactId>extentreports</artifactId>
			<version>2.41.2</version>
		</dependency>

![image](https://user-images.githubusercontent.com/24426914/33605486-e2b714ca-d96e-11e7-95f1-581a8910327a.png)

**Comment by email2vimalraj** [comment]
@rajay458 : Look like you are running with way older extent report. We have fixed and improved many things after v2.41.1. Kindly upgrade to the newer version (v3.1.1) and try.

**Comment by anshooarora** [comment]
PS.  you can use https://github.com/email2vimalraj/TestNGExtentsReport which will setup the structure and provide all boilerplate code.
